mathcomp/_CoqProject

```diff
@@ -5,3 +5,5 @@
 -arg -w -arg -redundant-canonical-projection
 -arg -w -arg -notation-overridden
 -arg -w -arg -duplicate-clear
+-arg -w -arg -ambiguous-paths
+-arg -w -arg -undeclared-scope
```

Comment:
Is this silencing "undeclared-scope" warnings? If so it is not completely clear to me that we want to do this rather than fixing it...

Reply:
Oh, after re-reading myself, it seems obvious that the reason is backward compatibility. My bad...